### DISCUSSION TITLE
9460C0DB-B497-E73B-DBB2-5697A56C7838

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#187] [General]
# 9460C0DB-B497-E73B-DBB2-5697A56C7838

The Horsehead Nebula (also known as Barnard 33, and IC 434) is a small dark nebula about 1,375 light-years away in the constellation Orion.

https://celestia.mobi/addon?item=9460C0DB-B497-E73B-DBB2-5697A56C7838

**Comment by EfremAlvares** [comment]
I don't know if this is still two dimensional or it's now it's three dimensional

**Reply by DaveBowman2001** [reply]
Unfortunately the current addon is still a two-dimensional billboard model originally created in 2008. Afaik no one has yet developed a fully 3D model of this nebula for Celestia
